pages/design/atomic-template-empty.md

```diff
@@ -50,7 +50,24 @@ _There are no major accessibility support issues known for this rule._
 
 ## Background
 
-- (e.g. WCAG Techniques or links with background information mentioned in Applicability, Expectations or Assumptions)
+### Understanding WCAG 
+
+- [WCAG 2.0 - Understanding 4.1.2: Name, Role, Value](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html)
+
+### Related WCAG Techniques
+
+- [G108: Using markup features to expose the name and role](https://www.w3.org/TR/WCAG20-TECHS/G108.html)
```

Comment:
Do we want "Related WCAG Techniques" here? 
If the rule is a test for a technique, it can be listed under Accessibility Requirements, where we'll get the benefit of the mapping too. 
If the related technique does not qualify to be put in the Accessibility Requirements section, should it then be listed on the rule at all?

Reply:
@WilcoFiers, I would like to discuss this. Do you have time any time soon?

Reply:
It could test part of a technique or could mention a technique in it's notes. In both cases it would be handy to link to the technique in this section I would say.